app/interactions/whois/update.rb

```diff
@@ -33,7 +33,11 @@ def determine_class
     end
 
     def find_record(klass, name)
-      klass == DNS::Zone ? klass.find_by(origin: name) : klass.find_by(name: name)
+      if klass == Dispute.active
```

Comment:
Since that's not a class anymore, but more of some scoped collection, I suppose we should rename this arg to something like `scoped_collection` just to not have a semantic confusion.
Same with `determine_class` function